pkg/controller/siddhiprocess/siddhiprocess_controller.go

```diff
@@ -120,7 +162,7 @@ type ReconcileSiddhiProcess struct {
 // and what is in the SiddhiProcess.Spec
 func (rsp *ReconcileSiddhiProcess) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
-	reqLogger.Info("Reconciling SiddhiProcess")
+	reqLogger.V(1).Info("Reconciling SiddhiProcess")
```

Comment:
What's this V(1). Is it how you set to debug level

Reply:
Yes, here we have used controller runtime log object as the logger inside the siddhi operator. So this is the way they recommended to made logger into the debug mode. For more details [refer this](https://github.com/kubernetes-sigs/controller-runtime/blob/master/TMP-LOGGING.md#getting-started-with-structured-logging)